app/assets/javascripts/discourse/widgets/post-menu.js.es6

```diff
@@ -301,6 +301,41 @@ registerButton("bookmark", attrs => {
   };
 });
 
+registerButton("bookmarkWithReminder", (attrs, state, siteSettings) => {
+  if (!attrs.canBookmark || !siteSettings.enable_bookmarks_with_reminders) {
+    return;
+  }
+
+  let className = "bookmark with-reminder";
```

Comment:
I think a cleaner pattern for classnames is to do this:

```
const classNames = ["bookmark", "with-reminder"];

if (foo) {
  classNames.push("bookmarked");
}

return classnames.join(" ");
```

Reply:
That is much better and more flexible, thank you